strax/context.py

```diff
@@ -904,6 +904,41 @@ def get_array(self, run_id: ty.Union[str, tuple, list],
             results = [x.data for x in source]
         return np.concatenate(results)
 
+    def get_accum_array(self,
```

Comment:
Maybe `get_accumulated`? A one-element array with datatypes is perhaps a bit inconvienent/odd: how about returning a dictionary instead?

Reply:
Hej Jelle, sorry for my late reply. Sounds great, 

> A one-element array with datatypes is perhaps a bit inconvienent/odd: how about returning a dictionary instead? 

Do you mean something like: `{'channel': np.array([1, 2, 3, 4, 5 ...]), 'dt': 10, ....}` or only in the case of a real one-element array like idk: `np.array([1,2,3,4,5,6], ('Single elements', 'bla'), dtype=np.int32)` 

>  How about passing the sum function as an argument, so the user could also do e.g. std or other simple reductions quickly?

I like that idea, will change it.